src/atemSwitcher.ino

```diff
@@ -151,6 +152,7 @@ void loop() {
 
     updateState();
     updateATEM();
+    muteMainAudio();
```

Comment:
This doesn't really describe the function, I'd say updateAudio or something. 'muteMainAudio' sounds like that is only what it does, whereas it can unmute it too.